templates/ts-apollo-fullstack/server/src/resolvers/resolvers.ts

```diff
@@ -9,85 +9,85 @@ export default {
   Note: {
     comments: (parent, args, context) => {
       return context.Comment.batchLoadData("noteId", parent.id, context)
-    }
+    },
```

Comment:
Invalid `,`

Reply:
Nothing in code has changed to make this happen, I investigated this and it appears this is down to the Prettier formatter:

https://github.com/aerogear/graphback/blob/9df2aa574299db464b7f77293bb493f1f799c522/packages/graphback-codegen-resolvers/src/templates/formatter.ts#L16

https://prettier.io/blog/2020/03/21/2.0.0.html

Prettier v2 has changed the default value for `trailingCommas` to 'all'

https://github.com/prettier/prettier/issues/68

Reply:
Is there any negative points to keeping it using the es5 standard? This is a generate file anyway.

Reply:
Knowing it was prettier we can ignore that